### DISCUSSION TITLE
Loading a file into an audio clip should/could be more accessible/intuitive.

### DISCUSSION THREAD

**Discussion by ok-reza** [SynthstromAudible/DelugeFirmware#2768] [General]
### This is the current official behavior

When you enter an newly created, blank audio clip, no sample is loaded. Since you can't use LOAD+AUDIO (since there isn't an AUDIO button like there is for SYNTH/KIT/MIDI) then the user must use SHIFT+BROWSE to enter the browser. Only until recently was a community feature added where an AUDIO CLIP menu was added (via pressing SEL encoder). From there you must first scroll a few notches down to SAMPLE to drill into the SAMPLE BROWSER. Both options are a little vague and less accessible for a less experienced user.

### Some possible improvements:

Would a good solution be: pressing **any** grid pad in an empty audio clip open up the sample browser?
> After a sample is loaded, would it be frustrating to make any grid pad press reopen the browser again (excluding the last right-most column, which opens the red end point column)? Or should it only work when the audio clip is empty, the user must use the aforementioned shortcut or menu to access the sample browser?

Should the SAMPLE item in the AUDIO CLIP menu be moved to the first on the list? This would make it easier to access the sample browser quickly.

Or should the sample browser automatically be open when entering a blank audio clip (no grid press necessary)? This could potentially be frustrating for some users that may just want to adjust Audio Clip parameters for a clip designated for audio monitoring, so maybe this isn't the right solution.

Should a "LOAD+SYNTH"-style shortcut also be enabled for audio clips. An example would be for ROWS view: LOAD+PRESS SEL to open the sample browser (since pressing SEL is essentially the AUDIO clip button).

Or is this a non-issue entirely?

**Comment by duncanrmi** [comment]
can we go back a step? something I've noted, with some irritation, in the past is that when I've done a load of live audio looping or recording, I end up with loads of files that I can't housekeep. I want to name some of them, bin some of them.... 

& then, yeah, absolutely, create a new audio clip & quickly load one of them, from a folder structure that should be the first one visible when you are in an audio clip. shift-browse would do to get that to happen; I don't think it currently does anything else useful in the context of an audio clip.

**Comment by RJ-Eckie** [comment]
I'm personally not too big of a fan when devices are trying to do too much thinking for me - the frustration of the device occasionally doing things that I don't want to do and me having to cancel out of it far outweighs the small effort of taking the steps it would take for me to reach that same behavior when I _do_ want it. I can do SHIFT+BROWSE with my eyes closed because of how close it is to the natural positions of my hands on the sides of the device. I also do use empty audio clips.

**Comment by bukru** [comment]
I my opinion this is not really a problem. SHIT+BROWSE it is ok for me : ] And i agree with RJ-Eckie
